src/main/kotlin/no/nav/faktureringskomponenten/metrics/MetrikkNavn.kt

```diff
@@ -0,0 +1,16 @@
+package no.nav.faktureringskomponenten.metrics
+
+class MetrikkNavn {
+    companion object {
+        val METRIKKER_NAMESPACE = "faktureringskomponenten"
+
+        val FAKTURASERIE_OPPRETTET = "${METRIKKER_NAMESPACE}.fakturaserie.opprettet"
+        val FAKTURA_BESTILT = "${METRIKKER_NAMESPACE}.faktura.bestilt"
+
+        // Blir lagt til når vi får statuser fra OEBS
```

Comment:
Kommentarene bør tas vekk i denne filen ettersom de er helt unødvendig for å kunne bruke metrics

Reply:
Hvis du tenker det er selvforklarende fjerner jeg dem 👍